### ISSUE TITLE
[FEATURE] - Add pulsar.listener.autoStart property

### ISSUE THREAD

**Issue by ThePotatoProgrammer** [majusko/pulsar-java-spring-boot-starter#156]
I am not sure if this is possible but my lead dev is requesting a property that will work like the Kafka listener autostart property. He doesn't want to always listen to pulsar messages when developing new features. Would it be possible to implement this?

`pulsar.listener.autostart=true` -> application works as it does right now with pulsar integration.
`pulsar.listener.autostart=false` -> same behavior as Kafka listener.autostart=false

**Comment by majusko** [comment]
Hi @ThePotatoProgrammer, I will need to do a bit of research on how would this be possible with Pulsar but if it's something doable, I can implement this for sure. D you have any resources on Kafka implementation? I can imagine there is a limitation from the Pulsar client on this but maybe it's just a library handling.

**Comment by ThePotatoProgrammer** [comment]
@majusko I have not worked with Kafka in the past, so I am not sure about the difficulty of implementation here. Sorry I cant be more helpful :'(

**Comment by majusko** [comment]
@ThePotatoProgrammer I will need to research this one and I will let you know if this is possible.

**Comment by majusko** [comment]
The implementation is ongoing and you will have the following options:

Setting global autostart configuration using `application.properties`: 

```
pulsar.auto-start=false
```

and also an option for configuring individual consumers:

```
 @PulsarConsumer(topic = "topic", clazz = MyMsg.class, utoStart = false)
```

**Comment by majusko** [comment]
The auto-start feature has been released in the 1.1.0 minor update.
